docs/ref/models/constraints.txt

```diff
@@ -19,6 +19,16 @@ option.
     convention is to use ``from django.db import models`` and refer to the
     constraints as ``models.<Foo>Constraint``.
 
+.. admonition:: Constraints in abstract base classes
+
+    You must always specify a unique name for the constraint. This would
+    normally cause a problem in abstract base classes, since the
+    :attr:`Meta.constraints <django.db.models.Options.constraints>` option on
+    this class are included into each of the child classes, with exactly the
+    same values for the attributes (including ``name``) each time. To work
+    around this problem, ``constraints`` option should be specified for each of
+    the child classes with unique names for constraints.
```

Comment:
I'd rephrase without `problem`...

```
    You must always specify a unique name for a constraint. As such, you
    cannot normally specify a constraint on an abstract base class, since the
    :attr:`Meta.constraints <django.db.models.Options.constraints>` option is
    inherited by subclasses, with exactly the same values for the attributes
    (including ``name``) each time. Instead, specify the ``constraints`` option
    on subclasses directly, providing a unique name for each constraint.
```